app/src/main/java/com/severianfw/picto/view/detail/PhotoDetailActivity.kt

```diff
@@ -25,13 +27,24 @@ class PhotoDetailActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         _binding = ActivityDetailBinding.inflate(layoutInflater)
         setContentView(binding.root)
+        setupIconColor()
 
         val photoItem = intent.getParcelableExtra<PhotoItemModel>(Constant.PHOTO_ITEM)
         setPhotoData(photoItem)
         setBackButtonListener()
         setDownloadButtonListener(photoItem)
     }
 
+    private fun setupIconColor() {
+        if (DarkModeUtil.isDarkModeActive(resources)) {
```

Comment:
I think we need to setup single source of truth of `isDarkModeActive`
Either we fetch from `DarkModeUtil` or `DarkModeRepository` 

As of now. I prefer `DarkModeUtil` 

So Dependency will be right this

```
class DarkModeUtil @Inject constructor (private val darkModeRepository: DarkModeRepository) {

}
```